module/zfs/vdev.c

```diff
@@ -2689,42 +2689,79 @@ vdev_accessible(vdev_t *vd, zio_t *zio)
  * Get statistics for the given vdev.
  */
 void
-vdev_get_stats(vdev_t *vd, vdev_stat_t *vs)
+__vdev_get_stats(vdev_t *vd, vdev_stat_t *vs)
```

Comment:
s/__vdev_get_stats/vdev_get_stats_impl()/

Am I correct in understanding this version must now be fully recursive because the collection of the stats is now done in the leaves whereas before it was at multiple levels of the tree?